src/components/Navbar/components/SearchBar/index.css

```diff
@@ -0,0 +1,35 @@
+.SearchBar {
```

Comment:
I personally don't think its great to have repetitive folders inside each other. `components/Navbar/components` where the last `components` folder could just be omitted IMO.

Then you just straight see the index.tsx, and its child components. without needing to open an additional folder.

Reply:
This also goes for the other files in this PR that have the same structure. :) Again, this is just an idea, not sure how the others think about it.

Reply:
I personally like this pattern because we can then add the `tests` folders for each component. Also if we have some need of adding another folder with another thing, like constants, another helper file with some functions that can be shared across those components, we do not mix them. So `components` is one folder, but then we can have other things. So, when the project grows and we start supporting other stores, with different behaviors, this will be necessary 😄